basic/js/userData.js

```diff
@@ -0,0 +1,8 @@
+export const USER_DATA = [
```

Comment:
```suggestion
export const users = [
```

USER_DATA처럼 UPPER_SNAKE_CASE로 사용하는 경우는 보통 상수인 경우에만 사용합니다(not const). user의 경우 변경될 수 있는 데이터이기 때문에 users로 하는게 조금 더 명확해보입니다. 또한, DATA라는 postfix는 쓸모가 없기 때문에 지양하는 편이 좋습니다.